network-api/networkapi/templates/pages/buyersguide/article_page.html

```diff
@@ -72,7 +72,7 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
             <div class="col-lg-8 tw-flex tw-py-4">
               {% if author_profile.image %}
                 <div class="tw-mr-4 tw-flex tw-items-start tw-flex-shrink-0">
-                  <img class="tw-rounded-full tw-border-2 tw-border-white" src="{% image_url author_profile.image "fill-75x75" %}" alt="{{ author_profile.name }}" title="{{ author_profile.name }}" />
+                  <img class="tw-rounded-full tw-border-2 tw-border-white tw-w-[75px] tw-h-[75px]" src="{% image_url author_profile.image "fill-225x225" %}" alt="{{ author_profile.name }}" title="{{ author_profile.name }}" />
```

Comment:
Should this be twice of 75 so `fill-150x150` instead?

Reply:
Let me know if the image still looks good after the change!